src/scraper/scraper.cpp

```diff
@@ -5002,7 +5002,7 @@ NN::Superblock ScraperGetSuperblockContract(bool bStoreConvergedStats, bool bCon
                     ConvergedScraperStatsCache.nTime = GetAdjustedTime();
                     ConvergedScraperStatsCache.Convergence = StructConvergedManifest;
 
-                    if (IsV11Enabled(nBestHeight)) {
+                    if (IsV11Enabled(nBestHeight + 1)) {
                         superblock = NN::Superblock::FromConvergence(ConvergedScraperStatsCache);
                     } else {
                         superblock = NN::Superblock::FromConvergence(ConvergedScraperStatsCache, 1);
```

Comment:
Is the reason for this that the scraper has to anticipate by one block the activation of v11? That is what this does when adding one. The scraper will start generating v2 SB contracts 1 one block before v11 activation, which seems correct, since this contract must PRECEDE v11 activation.

Reply:
Yes. For the unlikely case where a superblock occurs in the very first v11 block, this change ensures that the node that generates it does so in the new format.